protocol/v1/membership_update_message.go

```diff
@@ -448,6 +448,26 @@ func (g Group) Members() []string {
 	return g.members.List()
 }
 
+func (g Group) MemberPublicKeys() ([]*ecdsa.PublicKey, error) {
+	var publicKeys []*ecdsa.PublicKey
```

Comment:
It's a good idea to initialize with the capacity of `len(g.Members())`.

Reply:
Thanks, done